src/odemis/driver/smaract.py

```diff
@@ -2931,6 +2956,16 @@ def _doReference(self, future, axes):
                         logging.warning("Axis %s not referenced after the end of referencing", a)
                         # TODO: Raise some error here
 
+                    # if referenced, move to the safe posiiton (if defined)
+                    if self._pos_deactive_after_ref and self._deactive_pos_after_ref is not None \
```

Comment:
See my comment on the 5DoF concerning using the metadata directly + one variable.